### PR TITLE
AM-390: Jest tests for username change.

### PR DESCRIPTION
**Pull request by farmborough**
## :id: 
Fixes [AM-390](https://gravitee.atlassian.net/browse/AM-390)

## :pencil2: A description of the changes proposed in the pull request

Tests for [AM-389](https://gravitee.atlassian.net/browse/AM-389) but also some fixes for user organisation management endpoint and additional tests (organisation wasn't being tested at all for users).

All the files under `gravitee-am-test/api/management` are generated by the openapi tool and can be ignored.

[AM-390]: https://gravitee.atlassian.net/browse/AM-390?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AM-389]: https://gravitee.atlassian.net/browse/AM-389?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: gravitee-am-test/api/management/.openapi-generator/FILES [hunk 1]
```diff
@@ -152,6 +152,7 @@ models/NewUser.ts
 models/NotifierPlugin.ts
```
Comment (farmborough) [inline gravitee-am-test/api/management/.openapi-generator/FILES:152]: Auto generated, can be ignored.

File: gravitee-am-test/api/management/apis/AlertsApi.ts [hunk 2]
```diff
@@ -52,14 +52,14 @@ export interface GetAlertNotifierRequest {
     notifierId: string;
```
Comment (farmborough) [inline gravitee-am-test/api/management/apis/AlertsApi.ts:52]: This and every `ts` file in this `apis` directory is generated and can be ignored.

File: gravitee-am-test/api/management/models/AccessPolicy.ts [hunk 3]
```diff
@@ -74,16 +74,16 @@ export interface AccessPolicy {
     resource?: string;
     /**
      * 
-     * @type {Date}
+     * @type {number}
```
Comment (farmborough) [inline gravitee-am-test/api/management/models/AccessPolicy.ts:77]: These `models` files can also be ignored.

**Review by remisultan — DISMISSED** [review]
LGTM

**Review by remisultan — APPROVED** [review]

**Pull request merged**